core/parachain/pvf/session_params.hpp

```diff
@@ -9,27 +9,33 @@
 #include "runtime/runtime_api/parachain_host.hpp"
 #include "runtime/runtime_context.hpp"
 
+#include "parachain/pvf/runtime_params.hpp"
 namespace kagome::parachain {
-  inline outcome::result<runtime::RuntimeContext::ContextParams> sessionParams(
+  inline outcome::result<RuntimeParams> sessionParams(
       runtime::ParachainHost &api, const primitives::BlockHash &relay_parent) {
     // https://github.com/paritytech/polkadot-sdk/blob/e0c081dbd46c1e6edca1ce2c62298f5f3622afdd/polkadot/node/core/pvf/common/src/executor_interface.rs#L46-L47
     constexpr uint32_t kDefaultHeapPagesEstimate = 32;
     constexpr uint32_t kExtraHeapPages = 2048;
     OUTCOME_TRY(session_index, api.session_index_for_child(relay_parent));
     OUTCOME_TRY(session_params,
                 api.session_executor_params(relay_parent, session_index));
-    runtime::RuntimeContext::ContextParams config;
-    config.memory_limits.max_stack_values_num =
+    RuntimeParams config;
+    auto& contextParams = config.context_params;
+    contextParams.memory_limits.max_stack_values_num =
         runtime::RuntimeContext::DEFAULT_STACK_MAX;
-    config.memory_limits.heap_alloc_strategy =
+    contextParams.memory_limits.heap_alloc_strategy =
         HeapAllocStrategyDynamic{kDefaultHeapPagesEstimate + kExtraHeapPages};
     if (session_params) {
       for (auto &param : *session_params) {
         if (auto *stack_max = get_if<runtime::StackLogicalMax>(&param)) {
-          config.memory_limits.max_stack_values_num = stack_max->max_values_num;
+          contextParams.memory_limits.max_stack_values_num = stack_max->max_values_num;
         } else if (auto *pages_max = get_if<runtime::MaxMemoryPages>(&param)) {
-          config.memory_limits.heap_alloc_strategy = HeapAllocStrategyDynamic{
+          contextParams.memory_limits.heap_alloc_strategy = HeapAllocStrategyDynamic{
               kDefaultHeapPagesEstimate + pages_max->limit};
+        } else if (auto *pvfExecTimeout = get_if<runtime::PvfExecTimeout>(&param)) {
+          if (pvfExecTimeout->kind == runtime::PvfExecTimeoutKind::Backing) {
+            config.pvf_exec_timeout_backing_ms = pvfExecTimeout->msec;
+          }
```

Comment:
I think we can add approval timeout params to config as well

Reply:
Added Approval exec timeout as well